src/libraries/Microsoft.PowerFx.Connectors/Environment/PowerFxConfigExtensions.cs

```diff
@@ -90,5 +96,37 @@ public static IReadOnlyList<ConnectorFunction> AddActionConnector(this PowerFxCo
                 throw;
             }            
         }
+
+        public static async Task<ConnectorTableValue> AddTabularConnector(this PowerFxConfig config, string tableName, OpenApiDocument openApiDocument, IReadOnlyDictionary<string, FormulaValue> globalValues, HttpClient client, CancellationToken cancellationToken, ConnectorLogger configurationLogger = null)
+        {
+            return await config.AddTabularConnector(new ConnectorSettings($"_tbl_{tableName}"), tableName, openApiDocument, globalValues, client, cancellationToken, configurationLogger).ConfigureAwait(false);
+        }
+
+        public static async Task<ConnectorTableValue> AddTabularConnector(this PowerFxConfig config, ConnectorSettings connectorSettings, string tableName, OpenApiDocument openApiDocument, IReadOnlyDictionary<string, FormulaValue> globalValues, HttpClient client, CancellationToken cancellationToken, ConnectorLogger configurationLogger = null)
```

Comment:
>AddTabularConnector [](http://example.com/codeflow?start=54&length=19)

New API to allow Tabular connectors
Note that this code will only work for SQL connector as we hardcode some API function here
We'll need to identify how to generalize the function determination, for getting the schema & getting the items